buildspec-tests.yml

```diff
@@ -0,0 +1,18 @@
+version: 0.2
+
+phases:
+  install:
+    runtime-versions:
+      python: 3.9
+    commands:
+      - echo Installing dependencies...
+      - pip install -r requirements-test.txt
+  
+  build:
+    commands:
+      - echo Running Tests
+      - pytest
+  
+  post_build:
+    commands:
+      - echo Tests Complete
```

Comment:
Is there any actual value in this post build command?

Reply:
Probably not, since CodeBuild has its own `STATUS` anyway.